base-notebook/test/test_container_options.py

```diff
@@ -240,7 +240,6 @@ def test_container_not_delete_bind_mount(container, tmp_path):
     assert len(list(tmp_path.iterdir())) == 1
 
 
-@pytest.mark.skip(reason="not yet implemented; TODO: cherry-pick b44b7ab")
```

Comment:
I have read the test:
- `SECRET_LOVE` only appears once is this right?
- we should probably check this for non-root user as well?

Reply:
Hmm, I'm just looking carefully for the first time myself, and I also notice that `SECRET_ANIMAL` is never tested. I'm not sure about the intention of this test.

We should definitely cover the non-root user with a test since that's a separate execution thread in the script.

Reply:
First part is fixed